packages/main/src/MultiComboBox.js

```diff
@@ -572,17 +579,33 @@ class MultiComboBox extends UI5Element {
 		this._keyDown = false;
 	}
 
-	async _onkeydown(event) {
+	_onkeydown(event) {
+		const isArrowDownCtrl = isDownCtrl(event);
+
 		if (isShow(event) && !this.readonly && !this.disabled) {
 			event.preventDefault();
 			this.togglePopover();
 		}
 
-		if (isUp(event) || isDown(event)) {
-			this._handleArrowNavigation(event);
+		if (isDownShift(event) || isUpShift(event)) {
```

Comment:
When in the input field, if there is a value on Shift + Arrow Up we expect to not have anything (do nothing) but instead, the text is selected. The same is presented in UI5, so just check whether we can and have to prevent it.